pkg/receive/limiter_test.go

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package receive
+
+import (
+	"testing"
+)
```

Comment:
nit (curious that `fmt` does not pick this up :thinking: )
```suggestion
import "testing"
```